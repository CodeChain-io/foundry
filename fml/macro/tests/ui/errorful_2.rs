extern crate fml_macro;

#[fml_macro::service]
pub trait MyHandle: Service {
    fn fn_1(&self, a: i32, b: String) -> String;
    type What;
}

fn main() {

}