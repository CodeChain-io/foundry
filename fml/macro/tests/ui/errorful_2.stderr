error: Service trait must have only methods
 --> $DIR/errorful_2.rs:6:5
  |
6 |     type What;
  |     ^^^^^^^^^^
